adapters/mobfoxpb/params_test.go

```diff
@@ -37,6 +37,7 @@ func TestInvalidParams(t *testing.T) {
 
 var validParams = []string{
 	`{"TagID": "6"}`,
+	`{"key": "1"}`,
 }
 
 var invalidParams = []string{
```

Comment:
Please consider adding invalid param values for `key` param as well for the `invalidParams` test cases